src/frontend/screens/ObservationCreate/index.tsx

```diff
@@ -151,7 +163,7 @@ export const ObservationCreate = ({
     [addNewTrackLocations, addNewTrackObservation, value],
   );
 
-  const createObservation = React.useCallback(async () => {
```

Comment:
removing the top-level async to account for some of the errors that are thrown early in this function, which don't need to be wrapped with async. similar to what #1016 addresses, although in a different context